app/views/providers/home_address_lookups/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= form_with(
+      model: @form,
+      scope: :address_lookup,
+      url: providers_legal_aid_application_home_address_lookup_path(@legal_aid_application),
+      method: :patch,
+      local: true,
+    ) do |form| %>
+
+  <%= page_template(
+        page_title: t(".heading"),
+        template: :basic,
+        form:,
+      ) do %>
+
+    <%= form.govuk_fieldset legend: { size: "xl", tag: "h1", text: t(".heading") }  do %>
+      <%= form.govuk_text_field :postcode, label: { text: t(".label.postcode"), tag: "h2", size: "m" }, width: 10 %>
+      <%= form.govuk_text_field :building_number_name, label: { text: t(".label.building_number_name"), tag: "h2", size: "m" }, width: 10 %>
+      <%= govuk_warning_text(text: t(".helper_text")) %>
+    <% end %>
+
+    <p><%# govuk_link_to t(".address_manual_link"), providers_legal_aid_application_address_path %></p>
```

Comment:
Nits:
- Consider commenting the `<p>` tag, else there will visually be additional spacing on the page
- Consider adding `<p class="govuk-body">` to make use of the govuk styling when we enable the manual home link

Reply:
This was moved to a shared partial in a later commit - app/views/shared/_address_lookup.partial and the <p> tag and link are only rendered if the location == "correspondence" - I have added the govuk-body class to the <p> tag.